packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -192,7 +192,7 @@ export class QuickFileOpenService implements QuickOpenModel, QuickOpenHandler {
                 const first = sortedResults[0];
                 sortedResults.shift();
                 if (first) {
-                    const item = await this.toItem(first.getUri()!, { groupLabel: 'file results', showBorder: true });
+                    const item = await this.toItem(first.getUri()!, { groupLabel: 'file results', showBorder: !!recentlyUsedItems.length ? true : false });
```

Comment:
i guess there is no need to have `? true : false ` ?

Reply:
Ohh true! :see_no_evil:

Reply:
@elaihau updated, thank you!